Release/56.0.0 (#2581)
MetaMask/snaps: packages/snaps-rpc-methods/CHANGELOG.md

## [Unreleased]

## [10.0.0]
### Added

> - **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook which is meant to bind to the `addAndShowRequest` method of the `ApprovalController`. ([#2509](https://github.com/metamask/snaps/pull/2509))

Reviewer: ```suggestion
- **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook ([#2509](https://github.com/metamask/snaps/pull/2509))
   - It should bind to the `addAndShowRequest` method of the `ApprovalController`.
```

## [Unreleased]

## [10.0.0]
### Added
- **BREAKING:** `snap_dialog` now takes the `requestUserApproval` hook ([#2509](https://github.com/metamask/snaps/pull/2509))

>    - It should bind to the `addAndShowRequest` method of the `ApprovalController`.

Reviewer: Made a small mistake in my original suggestion. 😅
```suggestion
  - It should bind to the `addAndShowRequest` method of the `ApprovalController`.
```
